integration-tests-en/config.ini

```diff
@@ -3,4 +3,4 @@ ignored_result_keys = sender,recipient_id
 
 [protocol]
 type = rest
-url = http://localhost:5005/webhooks/rest/webhook
+url = %(CORE_ENDPOINT_URL)s/webhooks/rest/webhook
```

Comment:
c'est normal le `s` à la fin ici?

Reply:
Yes, it's python's `configparser` syntax for string interpolation